### ISSUE TITLE
Context Switches Every Time You Alt+Tab

### ISSUE THREAD

**Issue by HoldOffHunger** [RickStrahl/MarkdownMonster#1041]
To reproduce:

1. Open the search window in MarkDown Monster, i.e., control+f. Leave the text cursor in the search bar of the find window.
2. Alt+Tab into another window.
3. Alt+Tab back into MarkDown Monster.
4. BUG: Text cursor context is still in the search bar.  For ONE SECOND. Then it instantly switches back to the document if you start typing.  If you WAIT one second, then don't type, the problem of text cursor context switching doesn't happen.

TLDR: Alt+Tab out and back i nto MarkDown Monster when doing a search.

**Comment by HoldOffHunger** [comment]
I'm sorry!  I went through and made a mistake in my instructions, sorry.

NEW INSTRUCTIONS -- (try to do as fast as possible, with 1 second delay)

1. Open MarkDown Monster.
2. Alt+Tab into another window.
3. Alt+Tab back into MarkDown Monster.
4. Press control+f and try to search.
5. BUG: Text cursor is briefly in the search bar, for one second, and THEN INSTANTLY goes back to the text document.

Attached is video.


https://github.com/RickStrahl/MarkdownMonster/assets/14262061/6c4308a3-b6b8-46b8-a07f-74cc107b5d6a

**Comment by RickStrahl** [comment]
All that works for me. 

I'm not sure what you're showing in the video but it seems to have nothing to do with the steps you're describing?

Are you trying to prank me? 😂

**Comment by RickStrahl** [comment]
If anything it seems to me you are describing perhaps a focus issue where focus is not activating the editor quickly enough. I don't see that, but I suppose if you have a very slow machine it's possible that this can happen. MM's editor is a WebView control that has to be activated so it doesn't by default stay active when navigating to another app - there is explicit code that runs to return focus back. This should be quick and even on my slow machines I do not see this as a problem.

Maybe simplify the problem. When you come back to MM - where is the cursor? And is the cursor active (an inactive cursor is greyed out and not blinking).

**Comment by HoldOffHunger** [comment]
No pranks!

In the video --

Look at 0:17 -- The very bottom left, the text is "`r/TheFarLeftSide`".

Look at 0:19 -- I hit "Control+F", and hit shift+insert, to insert the text "`r/PluralLeft".

Look at 0:20 -- I hit "Enter" twice! I wanted to search for that text I just entered, instead, it creates two new lines!  "`/rTheFarLeftSide`" is NOW -- "`/rTheFarLeftSide(NEWLINE)(NEWLINE)`"

This is a bug, surely?

**Comment by HoldOffHunger** [comment]
Possibly my control+f and shift+insert are inadvertantly triggering some rare, hidden hotkey?

I have seen the problem more than 100 times. I had issues earlier describing/replicating, but now I have it down pat.

**Comment by HoldOffHunger** [comment]
If you agree that the context-switching from the search box to the document seemingly randomly is a bug, these are the weird conditions under which I seem to exclusively see it:

1. I am alt-f4'ing into the document.
2. I am very, very, very quickly pressing control+f.
3. Also equally quickly, I hit shift+insert.  I'm using text like "r/SomeRedditSub" and hitting enter.
4. Some microseconds after the above, context switches from the search bar to the document.

**Comment by HoldOffHunger** [comment]
I uninstalled and am now using a generic Markdown Editor/previewer. Thank you.
